generator/src/main/java/com/scottlogic/deg/generator/VisualiseExecute.java

```diff
@@ -56,15 +59,15 @@ public VisualiseExecute(DecisionTreeFactory profileAnalyser,
         this.configSource = configSource;
         this.outputPath = outputPath;
         this.profileReader = profileReader;
+        this.profileValidator = profileValidator;
         this.validator = validator;
     }
 
     @Override
     public void run() {
-
-        ValidationResult validationResult = validator.validateCommandLine(
-            configSource.overwriteOutputFiles(), outputPath);
-        if (!validationResult.isValid()) {
+        ValidationResult validationResult = validator.validateCommandLine(configSource.overwriteOutputFiles(), outputPath);
+        ValidationResult validationResult2 = profileValidator.validateProfile(configSource.getProfileFile());
```

Comment:
should we change the names of these to be more readable? i.e. `commandLineArgumentsResult` and `profileSchemaResult`